src/crypto/crypto-ops.h

```diff
@@ -30,6 +30,8 @@
 
 #pragma once
 
+#include <stdint.h>
```

Comment:
Why was it added here? `crypto-ops.c` already includes it before `crypto.h`

Reply:
It makes sense then to remove `#include <stdint.h>` from `crypto-ops.c`

Reply:
This cleans up static analyzer errors.

If you [google transitive includes](https://www.google.com/search?client=firefox-b-1-d&q=transitive+includes) you don't find anyone recommending they be used. Here is the somewhat authoritative [cpp core guideline](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rs-implicit).